dataland-frontend/src/components/forms/parts/elements/basic/ExtendedDataPointFormField.vue

```diff
@@ -178,12 +177,6 @@ export default defineComponent({
     showDataPointFields(): boolean {
       return this.dataPointIsAvailable || !this.isDataPointToggleable;
     },
-    isDataValueProvided(): boolean {
-      return (assertDefined(this.checkValueValidity) as (dataPoint: unknown) => boolean)(this.dataPoint);
-    },
-    isDataQualityRequired(): boolean {
-      return this.isDataValueProvided;
```

Comment:
So wie ich den Code in `main` verstehe, soll `quality` immer "ausgegraut" sein, wenn `value` nicht ausgefüllt ist.

D.h, wir möchten den user daran hindern, einen Wert für `quality` anzugeben, aber gleichzeitig keinen `value` mitzuschicken.

Wollen wir das nicht beibehalten?